doc/sphinx-guides/source/user/dataset-management.rst

```diff
@@ -123,6 +123,45 @@ There are several advanced options available for certain file types.
 - Image files: .jpg, .png, and .tif files are able to be selected as the default thumbnail for a dataset. The selected thumbnail will appear on the search result card for that dataset.
 - SPSS files: SPSS files can be tagged with the language they were originally coded in. This is found by clicking on Advanced Options and selecting the language from the list provided.
 
+.. _rsync_upload:
```

Comment:
Where it's not referent to data transfer, it might be more accurate to refer to this as DCM upload using rsync+ssh (or better wording to that effect).  rsync+ssh by itself doesn't do client-side checksums (which the DCM does), and by itself doesn't require a script.